### ISSUE TITLE
[1.6.0] Critical issue in fin 1.41.0. Unable to install Docksal from scratch.

### ISSUE THREAD

**Issue by 610** [docksal/docksal#422]
When trying to install docksal from scratch on a fresh macOS installation.
```
$ curl -fsSL https://get.docksal.io | sh
docker-machine executable was not found. (Try running 'fin update')
```
Just one error message printed and nothing else.

**Describe the results you received:**

Nothing is installed.

**Describe the results you expected:**

Docksal one line command installs everything is needed to run the docker and docksal.

**Comment by lmakarov** [comment]
Oops... Fixed in https://github.com/docksal/docksal/commit/3a39dbf51d314ab4e5050ee70c486a600906f980

Please confirm this fixes it for you and I'm going to roll out a hotfix release.

**Comment by achekulaev** [comment]
@610 try now installing develop version by `curl -L test.docksal.io | sh`

**Comment by lmakarov** [comment]
Fixed and rolled out in [v1.6.1](https://github.com/docksal/docksal/releases/tag/v1.6.1)
